src/rpc/rawtransaction.cpp

```diff
@@ -369,7 +369,7 @@ static UniValue gettxchainlocks(const JSONRPCRequest& request)
                 {RPCResult::Type::OBJ, "", "",
                 {
                     {RPCResult::Type::NUM, "height", "The block height"},
-                    {RPCResult::Type::BOOL, "chainlock", "Chainlock status for the block containing the transaction"},
+                    {RPCResult::Type::BOOL, "chainlock", "The state of the corresponding block ChainLock"},
```

Comment:
Nit: imo the original wording was easier to understand

Reply:
The phrase "The state of the corresponding block ChainLock" was [used](https://github.com/dashpay/dash/blob/8ea9c406558c14aab9deff57c6209e506c8f74d1/src/wallet/rpcwallet.cpp#L1405) [more](https://github.com/dashpay/dash/blob/8ea9c406558c14aab9deff57c6209e506c8f74d1/src/rpc/rawtransaction.cpp#L190) and seemed more concise.